eth_account/account.py

```diff
@@ -539,7 +558,9 @@ def set_key_backend(self, backend):
         self._keys = KeyAPI(backend)
 
     @combomethod
-    def sign_message(self, signable_message: SignableMessage, private_key):
+    def sign_message(self,
+                     signable_message: SignableMessage,
+                     private_key: Union[bytes, HexStr, int, keys.PrivateKey]) -> Any:
```

Comment:
Any i/o SignedMessage because of decorator

Reply:
same here:
```suggestion
                     private_key: Union[bytes, HexStr, int, keys.PrivateKey]) -> SignedMessage:
```
 then `return cast(SignedMessage, self._sign_hash(message_hash, private_key)` on line 607